src/platforms/javascript/common/performance/automatic.mdx

```diff
@@ -0,0 +1,139 @@
+---
```

Comment:
Should this page be a subpage of the "Enabling" page?

Reply:
IMHO it's enough of a key idea to deserve its own entry in the sidebar. It's also slightly different than enabling the package. Enablement is "how do I turn it on," whereas both this and the corresponding "manual" page are more "okay, now that it's on, how do I use it?"